src/components/PersonLink/PersonRow.tsx

```diff
@@ -0,0 +1,83 @@
+import cn from 'classnames';
+import { Link, useParams, useSearchParams } from 'react-router-dom';
+import { Person } from '../../types';
+
+interface Props {
+  person: Person;
+}
+
+enum PersonSex {
+  Male = 'm',
+  Female = 'f',
+}
+
+export const PersonRow: React.FC<Props> = ({ person }) => {
+  const {
+    name,
+    sex,
+    born,
+    died,
+    fatherName,
+    motherName,
+    mother,
+    father,
+    slug,
+  } = person;
+
+  const [searchParams] = useSearchParams();
+  const { slug: selectedPerson } = useParams();
+
+  return (
+    <tr
+      data-cy="person"
+      className={cn({ 'has-background-warning': slug === selectedPerson })}
+    >
+      <td>
+        <Link
+          to={{
+            pathname: `/people/${slug}`,
+            search: searchParams.toString(),
+          }}
+          className={cn({ 'has-text-danger': sex === PersonSex.Female })}
+        >
+          {name}
+        </Link>
```

Comment:
You have 3 similar links for person, mother and father. You can create a `PersonLink`